modules/ROOT/pages/6.6_Economic_Operator.adoc

```diff
@@ -324,7 +324,7 @@ _Table 36. Economic operator postal address, expected elements_
 |*cac:Country/cbc:IdentificationCode* |Code |1 |A code that identifies the country. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr92-012[_tbr92-012_].
 
-*Rule*: The country of the contracting body must always be specified. Compulsory use of the Code List link:{url-tree}/dist/cl/ods/[Country from EU Vocabularies, which is based on ISO.]  
+*Rule*: The country of the contracting body must always be specified. Compulsory use of the Code List link:{url-tree-tmp}/dist/cl/ods/[Country from EU Vocabularies, which is based on ISO.]  
```

Comment:
Country from EU Vocabularies, which is based on ISO.
I think this text should not be part of the link

Reply:
Updated link text as 'Country'.